python/ml4ir/applications/ranking/tests/test_features.py

```diff
@@ -55,12 +47,16 @@ def test_sequence_encoding(self):
         encoding_size = 512
         feature_info = {
             "feature_layer_info": {
-                "embedding_size": embedding_size,
-                "encoding_type": "bilstm",
-                "encoding_size": encoding_size,
+                "type": "numeric",
+                "fn": "get_sequence_encoding",
+                "args": {
+                    "embedding_size": embedding_size,
```

Comment:
can the test try and verify that embedding_size != encoding_size is ok?

Reply:
oh wait, it already is checking this, right?

Reply:
I think `flake8` has auto-formatted the code in some "weird" way. But yeah, in the test itself the encoding_size != embedding_size